scripts/pre_commit_linter.py

```diff
@@ -633,15 +634,35 @@ def _check_bad_patterns(all_files):
     return summary_messages
 
 
+def _check_import_order(all_files):
+    """This function is used to check that each file
+    has imports placed in alphabetical order.
+    """
+    print 'Starting import-order checks'
+    print '----------------------------------------'
+    all_files = [
+        filename for filename in all_files if not
+        any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)]
+    failed = False
+    for filename in all_files:
+        if isort.SortImports(filename, check=True).incorrectly_sorted:
+            failed = True
+    print ''
+    print '----------------------------------------'
+    print ''
+    return failed
+
+
 def main():
     all_files = _get_all_files()
+    import_order_message = _check_import_order(all_files)
     newline_messages = _check_newline_character(all_files)
     linter_messages = _pre_commit_linter(all_files)
     pattern_messages = _check_bad_patterns(all_files)
     all_messages = (
         linter_messages + newline_messages + pattern_messages)
     if any([message.startswith(_MESSAGE_TYPE_FAILED) for message in
-            all_messages]):
+            all_messages]) or import_order_message:
```

Comment:
why not just add import_order_message to all_messages? Also keep the order in line 663 the same as the order in lines 658-661 for consistency.

Reply:
import_order_message returns boolean and not the actual message. Therefore I used it this way. I will ensure the consistency though.